src/components/addbutton.js

```diff
@@ -59,7 +60,7 @@ export default class AddButton extends React.Component {
         this.hideBlock();
       } else {
         this.setState({
-          visible: true,
+          visible: true && hasFocus,
```

Comment:
`true && hasFocus` does not make sense

Reply:
oh, woops. Fair point, this makes no sense. I've pushed an updated version.